libraries/botbuilder-ai/package.json

```diff
@@ -35,18 +36,18 @@
     "lodash": "^4.17.21",
     "node-fetch": "^2.6.7",
     "url-parse": "^1.5.9",
-    "zod": "~1.11.17"
-  },
-  "resolutions": {
-    "follow-redirects": "1.14.7"
-  },
-  "devDependencies": {
+    "zod": "~1.11.17",
     "@types/node-fetch": "^2.5.7",
     "botbuilder-core": "4.1.6",
     "botbuilder-dialogs": "4.1.6",
     "fs-extra": "^7.0.1",
     "nock": "^11.9.1"
   },
+  "resolutions": {
+    "follow-redirects": "1.14.7"
+  },
+  "devDependencies": {
```

Comment:
Did you need to move all devDependencies?

Reply:
Fixed by moving botbuilder-core and botuilder-dialogs only